client/src/containers/profile/index.tsx

```diff
@@ -22,6 +22,8 @@ export const Profile: React.FC = () => {
 		switch (activeTabId) {
 			case ActiveTabId.Stats:
 				return <Stats statsInfo={statsProps} />;
+			case ActiveTabId.Tasks:
+				return <ProfileTasks profileTasks={profileTasks} />;
```

Comment:
Let's make a separate container for this tab, same as Stats. This will make it more convenient to work with logic. This container will contain the actions and connections to the redux store that are needed for this tab. Also, it is better to transfer mocks to this container too, so that each tab has a separate file with mocks.